### PR TITLE
Autostart daemon possible to give in a directory /home/pi/{directory}…

### PR DESCRIPTION
**Pull request by dannybarake**
…/dash

## Description:
Added an autostartdaemon for RPI. Shutdown and restart should be done otherwise for example with mausberry or something like that.

**Related issue (if applicable):** fixes #<issue number goes here>
Fix for issue #21 

## Checklist:
  - [X] The code change is tested and works locally.

### CODE REVIEW

File: src/bash/autostartdaemon.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+###
+WorkingDirectory="/home/pi/dash/"
+if [[ $1 != "" ]]
+then
+    WorkingDirectory="/home/pi/$1/dash/"
+fi
+echo ${WorkingDirectory}
+
+#stop and disable dash service
+sudo systemctl stop dash.service
```
Comment (rsjudka) [inline src/bash/autostartdaemon.sh]: does this work if there was no dash service previously?

File: rpi.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #location of OS details for linux
 OS_RELEASE_FILE="/etc/os-release"
 
-#check if Raspian OS is active, otherwise kill script
+## check if Raspian OS is active, otherwise kill script
```
Comment (rsjudka) [inline rpi.sh]: super comment? :p

File: src/bash/autostartdaemon.sh (hunk 1 above)
Comment (dannybarake) [reply src/bash/autostartdaemon.sh]: @rsjudka Yes, doesn't throw any errors

Comment (rsjudka) [reply src/bash/autostartdaemon.sh]: this is what i see on a system with no dash service:
```
$ sudo systemctl stop dash.service
Failed to stop dash.service: Unit dash.service not loaded.
```

File: rpi.sh (hunk 2 above)
Comment (dannybarake) [reply rpi.sh]: removed it :-)

**Comment by dannybarake** [comment]
@rsjudka Changed the script to prevent error when service doesn't exists

**Pull request merged**